src/main/java/seedu/address/storage/JsonAdaptedBuyer.java

```diff
@@ -50,7 +65,14 @@ public Buyer toModelType() throws IllegalValueException {
             throw new IllegalValueException(Price.MESSAGE_CONSTRAINTS);
         }
         final Price modelPrice = new Price(maxPrice);
-        return new Buyer(modelPerson, modelPrice);
+
+        final List<Tag> buyerTags = new ArrayList<>();
```

Comment:
Minor question. Is it necessary to have a List here instead of just using the Set? Alternatively, a map operation could be done for lines 70-72.